app/models/sac_cas/event/course.rb

```diff
@@ -28,9 +28,12 @@
 #  link_participants                :string(255)
 #  link_survey                      :string(255)
 #  location                         :text(65535)
+#  minimum_participants             :integer
 #  maximum_participants             :integer
 #  minimum_age                      :integer
-#  minimum_participants             :integer
+#  maximum_age                      :integer
```

Comment:
diese werte sind ja auf der db ... idealerweise müsstest du diese auch in SacCas::Event ergänzen.
oft wird die info auch noch in den fixture files hinterlegt.
aber ich mag das eigentlich auch nicht so und es wurde beim SAC wagon kaum gemacht